packages/reports/src/Components/Form/CompositeCalendar/composite-calendar-mobile.tsx

```diff
@@ -1,11 +1,19 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Button, DatePicker, Icon, InputField, MobileDialog, Text } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { toMoment } from '@deriv/shared';
 
-export const RadioButton = ({ id, className, selected_value, value, label, onChange }) => {
+type TRadioButtonProps = {
```

Comment:
need to change file name in the `packages/reports/src/Components/Form/CompositeCalendar/composite-calendar.jsx`

Reply:
@mahdiyeh-deriv I wanted to change it directly in composite-calendar component, since I'm working on that as well, but I can change it here too